examples/basic/empiricalrc.json

```diff
@@ -3,30 +3,14 @@
   "runs": [
     {
       "type": "model",
-      "provider": "openai",
```

Comment:
Would prefer reverting this change - basic example should have OpenAI models since they are the most common

Reply:
Reverting, this was a testing change